stories/topics/con-gcp-restore-process.adoc

```diff
@@ -8,8 +8,7 @@ The restore process deploys a new deployment, and restores the filestore and SQL
 =====
 * You must restore with the same operational image version as the backup.
 
-* You can restore using either a new VPC network or an existing one.
-You must configure additional parameters to restore on an existing VPC network.
+* You can only restore using either a new VPC network. We do not currently support backup and restore into an existing VPC network.
 
 * You cannot delete the backed up environment before a restore because the database backups are stored in the deployment.
```

Comment:
@zkayyali812 @itdove:

at this time, this note, though crucial for restore is added among several other important notes in the restore section. In my  opinion, this must be added to the backup process section after user performs backup, so user remembers not to delete the deployment that was backed up

Reply:
Great suggestion @pavankrao, I added this note as well as the other important notes.